pasqal_cloud/batch.py

```diff
@@ -149,6 +150,25 @@ def add_jobs(
                 time.sleep(RESULT_POLLING_INTERVAL)
                 self.refresh()
 
+    def retry(self, job: Job, wait: bool = False) -> None:
```

Comment:
Why doesn't it return the retried job or the whole batch ? What is the point of the wait in that case ?